src/cmd/manage_nsfs.js

```diff
@@ -971,6 +972,31 @@ function validate_boolean_string_value(value) {
     return false;
 }
 
+/**
+ * validate_min_flags_for_update validates that we have at least one flag of a property to update
+ * @param {string} type
+ * @param {object} input_options_with_data
+ */
+function validate_min_flags_for_update(type, input_options_with_data) {
+    const input_options_for_update = { ...input_options_with_data };
+    // remove the config options
+    delete input_options_for_update.config_root;
```

Comment:
I would use filter or difference of sets

Reply:
I changed it to filter, thanks.
After handling the GAP the function will be shorter and cleaner.